A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -299,6 +301,38 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
 
         }
 
+        // ECAM all button
+        this.ecamAllButtonState = SimVar.GetSimVarValue("L:A32NX_ECAM_ALL_Push_IsDown", "Bool");
+        const ECAMPages = [
+            "ENG",
+            "BLEED",
+            "PRESS",
+            "ELEC",
+            "HYD",
+            "FUEL",
+            "APU",
+            "COND",
+            "DOOR",
+            "WHEEL",
+            "FTCL",
+            "STS"
+        ];
```

Comment:
On line 274 u have const ECAMPageIndices 
Could you take it out of the if and use a single variable instead of initializing another?

```suggestion
```


Wouldn't it be better to create a function inside the BaseEICAS class that returns an array of pages, in order to make everything more automatic in case of changes? I ask!

Reply:
I see what you mean. The ECAMPageIndices at 274 wasn't put there by me, but I'll tidy things up to use the lowerScreenPages

Reply:
If you want, you can also wait for an answer from someone more experienced than me!
Without going to change your code

Reply:
I changed it. As far as I can tell, everything still works as it used to

Reply:
Well!
I think it's better, although unlikely, if pages are changed or added, your script will continue to work without further changes